Bash_scripts/HW21/in_hw20/roles/hw201/tasks/docker.yml

```diff
@@ -0,0 +1,43 @@
+#Ubuntu
+- name: "Install Docker on Ubuntu"
+  block:
+    - name: "Set up the repository"
+      ansible.builtin.package:
+       name: "{{item}}"
+       update_cache: yes
+      loop:
+       - apt-transport-https=2.0.5
```

Comment:
Package name and version should be two different variables

Reply:
done
https://i.imgur.com/Agw2UFI.png

Reply:
https://i.imgur.com/zH4GdHn.png